shakenfist/virt.py

```diff
@@ -290,9 +291,11 @@ def block_devices(self):
         return self._db_get_attribute('block_devices')
 
     # Implementation
+    def get_lock_attr(self, name, op):
```

Comment:
Sorry, brain fart. Shouldn't this be in baseobject.DatabaseBackedObject?

Reply:
Good point. The PR auto closed whilst I was messing with my personal develop branch. Will reopen.